### PR TITLE
web/satellite:access grant empty state

### PR DESCRIPTION
**Pull request by mbouzi**
What: 
Page state where there're no access grants
Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue [hunk 1]
```diff
@@ -22,6 +22,9 @@
             >
                 <p class="project-dropdown__wrap__choice__unselected">{{ project.name }}</p>
             </div>
+						<div @click="toggleSelection" @click.prevent.stop="closeDropdown" class="project-dropdown__wrap__create-project__wrapper">
```
Comment (VitaliiShpital) [inline web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue:25]: this change is here again. It should be made in a separate PR

File: web/satellite/src/components/accessGrants/EmptyState.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
+
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+<template>
+	<div class="empty-state">
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/EmptyState.vue:5]: seems like indents are broken here

File: web/satellite/src/components/accessGrants/EmptyState.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		<div class="empty-state__modal">
+			<Key />
+			<h4 class="empty-state__modal__heading">Create Your First Access Grant</h4>
+			<p class="empty-state__modal__subheading">Get started by creating an Access to interact with your Buckets</p>
+			<VButton
+				label="Create Access Grant +"
+				width="199px"
+				height="44px"
+				v-bind:style="{ margin: `0 auto`, marginTop: `25px`}"
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/EmptyState.vue:14]: could be simplified like this `:style="{ margin: `0 auto`, marginTop: `25px`}"`

I mean, it's not necessary to use v-bind

and by the way, I would not use inline styles if it's not necessary. You could give a class name to a component and style it as a regular tag inside style tags

**Review by VitaliiShpital — COMMENTED** [review]
did you compress png that is used as a background-image? I believe it can be compressed. 72Kb is too much

File: web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue (hunk 1 above)
Comment (mbouzi) [reply web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue:25]: sorry didn't mean for this to get in here
